.travis.yml

```diff
@@ -32,6 +39,8 @@ script:
   - python -m pytest --cov
   - codecov
   - docker run --volume=$(pwd):/app --workdir=/app coala/base coala-ci
+  - python3 setup.py install
+  - bash ./.ci/deploy.sh
```

Comment:
this should be in `deploy:`

Reply:
That’s a workaround for now, I’ll do that once tests start passing(as noted in PR description).